windows/winrm-gen.ps1

```diff
@@ -0,0 +1,83 @@
+param (
+    [string]$ComputerName = "servername.domain.com"
```

Comment:
To make it more useful, we can also default to the hostname of the Windows Machine the script will run on. Try:

```powershell
param (
    [string]$ComputerName = (hostname)
)
```